timelock-agent/src/main/java/com/palantir/timelock/config/TsBoundPersisterRuntimeConfiguration.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.timelock.config;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
+@JsonSubTypes(@JsonSubTypes.Type(value = DatabaseTsBoundPersisterRuntimeConfiguration.class, name = "database"))
```

Comment:
Why the subtyping?

Reply:
we discussed this offline:

>  re: subtyping, I didn’t want to have an optional block for kvs runtime config specifically (because we might end up needing a different optional block for runtime configs for other persistence modes in the future)